src/app/cmdoptions.cpp

```diff
@@ -54,6 +56,7 @@ namespace
     const int USAGE_INDENTATION = 4;
     const int USAGE_TEXT_COLUMN = 31;
     const int WRAP_AT_COLUMN = 80;
+    const QString DEFAULT_PORTABLE_MODE_PROFILE_DIR = QStringLiteral("profile");
```

Comment:
There is src/base/profile.cpp file so this declaration should not be here.

Reply:
Added this because there is same piece here: https://github.com/qbittorrent/qBittorrent/blob/master/src/app/application.cpp#L118

Should I try to move it all to src/base/profile.cpp ?

Reply:
>Should I try to move it all to src/base/profile.cpp ?

Maybe yes, at least it should try to fit in the helper functions that file provides. Let us wait for others opinion.